src/Ankimon/ankimon_items_web/settings.css

```
@@ -753,3 +753,20 @@
 
+/* ============================================================
+   Password input field styling
+   ============================================================ */
+.setting-input[type="password"] {
+    font-family: 'JetBrains Mono', monospace;
+    letter-spacing: 2px;
+}
+
+.setting-input[type="password"]::placeholder {
+    letter-spacing: 0px;
+    font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)